coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,42 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_top_files_block(self):
-        # TODO: rename once implemented.
-        assert 0
+    def test_response_content(self, rf):
+        """Test the response contains the url and parsed paths from the mocked file handle."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            b'192e635b17a9c2aea6181f0f87cab05d  data/file01.txt',
+            b'18b7c500ef8bacf7b2151f83d28e7ca1  data/file02.txt',
+            b'']
+        bag = FullBagFactory.create()
+        request = rf.get('/')
+        response = views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_path_is_not_unicode_safe(self):
-        assert 0
+        assert (b'https://coda/data/file02.txt\n'
+                b'https://coda/data/file01.txt') in response.content
+        assert response.status_code == 200
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_returns_status_code_200(self):
-        assert 0
+    def test_response_content_has_topFiles(self, rf):
+        """Test topFiles are returned in the response."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            b'<html coda/bag/123/manifest-md5.txt',
+            b'<html coda/bag/123/bagit.txt',
+            b'']
```

Comment:
This one could be changed to match the previous test.

Reply:
well...I didn't change it 'cos this test is written to test top files part of the code where we have `topFileHandle = getFileHandle(identifier, "")`. So, here we aren't opening the maifest.txt file.

Reply:
If that were the case, than this would not be in the response, right?
```
                b'https://coda/coda/bag/123/bagit.txt\n'
                b'https://coda/coda/bag/123/manifest-md5.txt') in response.content
```

Reply:
They are coming from mocked `getFileList`. I think I can change `bag-info.txt` to be `manifest-md5.txt`.